client/src/components/ElementEditor/ElementEditor.js

```diff
@@ -19,55 +19,69 @@ export const ElementEditorContext = createContext(null);
  * The ElementEditor is used in the CMS to manage a list or nested lists of
  * elements for a page or other DataObject.
  */
-class ElementEditor extends PureComponent {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      dragTargetElementId: null,
-      dragSpot: null,
-      elements: null,
-      isLoading: true,
-      dragging: false,
-    };
-
-    this.handleDragStart = this.handleDragStart.bind(this);
-    this.handleDragEnd = this.handleDragEnd.bind(this);
-    this.fetchElements = this.fetchElements.bind(this);
-  }
+const ElementEditor = ({
```

Comment:
The following props were added - previously referred to via this.props though not in propTypes:
- forceRefetchElements